dojo/tools/dependency_check/parser.py

```diff
@@ -358,3 +351,35 @@ def get_findings(self, filename, test):
 #     <software>cpe:2.3:a:oracle:rapid_planning:12.1:*:*:*:*:*:*:*</software>
 #     <software>cpe:2.3:a:netapp:snap_creator_framework:-:*:*:*:*:*:*:*</software>
 # </vulnerableSoftware>
+
+
+# TODO relateddependencies are ignored in this parser, but should be imported because you might miss vulnerable dependencies otherwise
```

Comment:
Move this comment in the comment of the parser

Reply:
I feel like this TODO is done already since the parser is importing related dependencies and tagging them as related in DD. I'll add a test for the related tag. Are you OK for me to remove this TODO comment blob?

Reply:
It looks like I already did add a test for the related tags. :) Line 734 in dojo/unittests/tools/test_dependency_check_parser.py